lib/fprotect/fprotect_rram.c

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdint.h>
+#include <string.h>
+#include <errno.h>
+#include <soc.h>
+#include <nrf_erratas.h>
+#include <hal/nrf_rramc.h>
+
+#define RRAMC_REGION_UNTOUCHED 0x0000000f
```

Comment:
Do I understand correctly this is a value of region 4 after a mass erase is performed? If that's the case how about renaming it to RRAMC_REGION_FOR_FPROTECT_DEFAULT_VALUE , _UNPROGRAMMED, _UNTOUCHED, or something like this